janusgraph-dist/pom.xml

```diff
@@ -13,6 +13,7 @@
 
     <properties>
         <top.level.basedir>${project.basedir}/..</top.level.basedir>
+        <cassandra-dist.version>4.0.3</cassandra-dist.version>
```

Comment:
Why did you add a specific `cassandra-dist.version` instead of upgrading the `cassandra.version`? This now means that we use `cassandra.version` (which is still `3.11.10`)  for tests, but `cassandra-dist.version` (`4.0.3`) in the zip distribution.
Shouldn't we always use the same version for our tests of the CQL backend as the one we include in the distribution?

Reply:
I will cleanup those variables, if i have removed the `cassandra-all` dependency.